editor/editor_node.cpp

```diff
@@ -590,7 +590,11 @@ void EditorNode::_update_theme(bool p_skip_creation) {
 		editor_main_screen->add_theme_style_override(SceneStringName(panel), theme->get_stylebox(SNAME("Content"), EditorStringName(EditorStyles)));
 		bottom_panel->add_theme_style_override(SceneStringName(panel), theme->get_stylebox(SNAME("BottomPanel"), EditorStringName(EditorStyles)));
 		distraction_free->set_button_icon(theme->get_icon(SNAME("DistractionFree"), EditorStringName(EditorIcons)));
-		distraction_free->add_theme_style_override(SceneStringName(pressed), theme->get_stylebox(CoreStringName(normal), "FlatMenuButton"));
+		if (distraction_free->get_meta("_scene_tabs_owned")) {
+			distraction_free->add_theme_style_override(SceneStringName(pressed), theme->get_stylebox(CoreStringName(normal), "FlatMenuButton"));
+		} else {
+			distraction_free->add_theme_style_override(SceneStringName(pressed), theme->get_stylebox(SceneStringName(pressed), "FlatMenuButton"));
+		}
```

Comment:
You could extract it into a method, because right now the style is applied in 2 different places.

Reply:
I suppose I should expose a public method in the `EditorNode` class to update the theme for this button, right? Or look for some signal to bind to this button (not sure which one though)